CMakeLists.txt

```diff
@@ -98,11 +98,9 @@ set(top_srcdir "${CMAKE_CURRENT_SOURCE_DIR}")
 configure_file(doc/manpage.rst.in doc/manpage.rst @ONLY)
 configure_file(doc/help.rst.in doc/help.rst @ONLY)
 
-configure_file(run_tests.sh.in run_tests.sh @ONLY)
-set(RE2C_RUN_TESTS "${CMAKE_CURRENT_BINARY_DIR}/run_tests.sh")
-if(CMAKE_HOST_UNIX)
-    execute_process(COMMAND chmod +x "${RE2C_RUN_TESTS}")
-endif()
+find_package(Python3 REQUIRED COMPONENTS Interpreter)
```

Comment:
I not familiar with this technique in autotools. Could you please send a commit to this branch?

Reply:
In Autoconf that would be something like `AC_CHECK_PROG(python, ...)`. But now I thought better of it and decided not to do it, because I still want re2c to build even if there is no python (or python3). The user may not be able to run tests, but this may be ok (e.g. if re2c is a dependency that they have to build on some weird platform).

Reply:
This line still bothers me.  I think package maintainers who use CMake may have unexpected issues

Reply:
I think it is fine --- a system that has CMake is likely to have python as well. The other option would be to drop REQUIRED and add a condition if (PYTHON3_FOUND) or some such when using python.

Autotools is a different story, because one can just unpack the release tarball (without the need to regenerate the configure script), and avoid any dependencies except for shell and C++ compiler. Not that I expect re2c to be used on systems without python, but I don't see any use in enforcing it either.

Reply:
(sorry, I accidentally edited your comment and then restored it back)